pdata/ptrace/generated_traces.go

```diff
@@ -620,14 +620,9 @@ func (ms Span) SetSpanID(v pcommon.SpanID) {
 	ms.getOrig().SpanId = data.SpanID(v)
 }
 
-// TraceState returns the tracestate associated with this Span.
-func (ms Span) TraceState() TraceState {
-	return TraceState(ms.getOrig().TraceState)
-}
-
-// SetTraceState replaces the tracestate associated with this Span.
-func (ms Span) SetTraceState(v TraceState) {
-	ms.getOrig().TraceState = string(v)
+// TraceStateStruct returns the tracestatestruct associated with this Span.
+func (ms Span) TraceStateStruct() pcommon.TraceState {
```

Comment:
This name is going to change in the future, right?

Reply:
Correct will become `TraceState`, but to do deprecation cannot reuse the same name.